StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -493,22 +479,3 @@ extension STPPaymentMethodParams {
         }
     }
 }
-
-extension Array where Element == PaymentSheet.PaymentMethodType {
```

Comment:
the original commit is here:
https://github.com/stripe/stripe-ios/commit/8e453258ec5750734432047e257b3615e90e238b
it looks like, we added `Hashable` to `PaymentSheet. PaymentMethodType` -- i wonder if we can get rid of `Hashable` ?

Reply:
This is still needed for `static var analyticLogForIcon: Set<PaymentMethodType> = []` - I won't try removing that in this PR.